### ISSUE TITLE
Тип описанных, но не инициализированных переменных не определяется при автогенерации документирующего комментария.

### ISSUE THREAD

**Issue by DmitryShvaika** [1C-Company/1c-edt-issues#1256]
### Описание ошибки

Тип описанных, но не инициализированных переменных не определяется при автогенерации документирующего комментария.

![image](https://github.com/1C-Company/1c-edt-issues/assets/54451084/115de99b-9ddf-4d8d-9100-ad22bd6ff858)

### Как воспроизвести

1. В общем модуле набрать текст:

```
// @strict-types

#Область ПрограммныйИнтерфейс

// Новый конструктор.
// 
// Возвращаемое значение:
//  Структура - Новый конструктор:
// * Реквизит 
Функция НовыйКонструктор() Экспорт

	Перем ЗначениеПустое; // Характеристика.МойПлан

	МояСтруктура = Новый Структура;
	МояСтруктура.Вставить("Реквизит", ЗначениеПустое);
	
	Возврат МояСтруктура;
	
КонецФункции

#КонецОбласти

```

2. Наблюдать ошибку

### Ожидаемое поведение

Тип реквизита должен определяться

### Лог рабочей области

не требуется

### Версия 1С:EDT

Ruby 2023.2.2

### Операционная система

Windows

**Comment by MaksimDzyuba** [comment]
Здравствуйте, а зачем вообще нужно

Перем ЗначениеПустое; // Характеристика.МойПлан

объясните, пожалуйста,  я просто не понимаю, смысл, если мы ее потом не инициализируем, почему просто не добавлять поле в структуру, а типизировать уже в комментарии, если на это ругается strict-types, то надо исправлять валидацию, как по мне

**Comment by DmitryShvaika** [comment]
Да, про документирующий комментарий и идет речь. Не добавлять поле в структуру нельзя, т.к. это функция-конструктор, - все поля должны быть определены в ней. Но если я определил тип в Перем, почему Edt не понимает этого?
